docs/api-docs/getting-started/rest-api-authentication.md

```diff
@@ -87,6 +87,7 @@ Where both types of API account are supported, review the preceding sections to
 | API or Use Case | App API Account | Store API Account |
 |:----------------|:---------------:|:-----------------:|
 | [REST Store Management APIs](/api-docs/getting-started/authentication#access-tokens) | &times; | &times; |
+| [GraphQL Store Management APIs](/api-docs/getting-started/authentication#access-tokens) | &times; | &times; |
```

Comment:
Need to confirm that the GraphQL DataLayer API authenticates using Store API Accounts (like the management APIs that are REST APIs)

Reply:
See comment [here](GraphQL DataLayer API (management API) authenticates using Store API Accounts: see comment [here](https://github.com/bigcommerce/api-specs/pull/1137/files#r1089433930). )